### ISSUE TITLE
Dropdown not closing other dropdowns

### ISSUE THREAD

**Issue by 57even** [ianmartorell/meteor-accounts-ui-bootstrap-3#136]
If I also have a dropdown next to the `meteor-accounts-ui-bootstrap-3` dropdown, it causes problems when I have the other dropdown active and then clicks the `meteor-accounts-ui-bootstrap-3` dropdown since the other dropdown wont close.

![dropdown](https://cloud.githubusercontent.com/assets/4533204/8406808/7d9070a8-1e60-11e5-9642-4e21f36007dc.png)

**Comment by ianmartorell** [comment]
It sounds to me like the problem is with your other dropdown. Does the opposite (having this package's dropdown open and clicking on the other one) work well?

**Comment by daoli** [comment]
I had the same problem. Removing these lines in ``login_buttons_dropdown.js`` resolves the issue.

```
-               },
-               'click .dropdown-toggle': function(event) {
-                       event.stopPropagation();
-                       Template._loginButtons.toggleDropdown();
```

**Comment by ianmartorell** [comment]
You're right, that was in place from when I tried to fix #147. This will be fixed in 1.2.74 when I publish in a few minutes
